rclpy/test/test_task.py

```diff
@@ -203,32 +209,32 @@ def test_executing(self) -> None:
 class TestFuture(unittest.TestCase):
 
     def test_cancelled(self) -> None:
-        f = Future()
+        f: Future[Any] = Future()
         f.cancel()
         self.assertTrue(f.cancelled())
 
     def test_done(self) -> None:
-        f = Future()
+        f: Future[None] = Future()
         self.assertFalse(f.done())
         f.set_result(None)
         self.assertTrue(f.done())
 
     def test_set_result(self) -> None:
-        f = Future()
+        f: Future[str] = Future()
         f.set_result('Sentinel Result')
         self.assertEqual('Sentinel Result', f.result())
         self.assertTrue(f.done())
 
     def test_set_exception(self) -> None:
-        f = Future()
-        f.set_exception('Sentinel Exception')
+        f: Future[Any] = Future()
+        f.set_exception('Sentinel Exception')   # type: ignore[arg-type]
```

Comment:
This is kind of abusing the method/type (using a `str` instead of an `Exception` like some of the other tests do), but for the purposes of this test it's fine.